app/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,7 @@
     <string name="cancel">Cancel</string>
     <string name="fdroid_vlc_url" translatable="false">https://f-droid.org/repository/browse/?fdfilter=vlc&amp;fdid=org.videolan.vlc</string>
     <string name="open_in_browser">Open in browser</string>
+    <string name="copy_url">Copy URL</string>
```

Comment:
Oh, sorry, forgot to say you should also remove this string, then the PR can be merged.

@opusforlife2 this PR only has to do with a button in the player dropdown menu, so no tooltips here